### PR TITLE
Link to Store Manager to clean out logs

### PR DESCRIPTION
**Pull request by scottcwilson**
Provide a link in the widget to make it easy to clean everything out.

### CODE REVIEW

**Comment by zcwilt** [comment]
:+1:

**Comment by drbyte** [comment]
👍 
Note: If the user doesn't have permission to access the Store Manager page, they'll get a "Denied" page if they click that link.

**Comment by ajeh** [comment]
Admin should be using zen_admin_href_link vs zen_href_link

**Comment by drbyte** [comment]
True.

**Comment by ajeh** [comment]
:+1:

File: includes/library/zencart/DashboardWidget/src/Logs.php [hunk 1]
```diff
@@ -59,6 +59,9 @@ public function prepareContent()
     }
     $tplVars['content'][] = array('text'=> $final_message, 'value'=> '');
 
+    $clean_message = '<a href="' . zen_href_link(FILENAME_STORE_MANAGER) . '">' . TEXT_CLEANUP_LOGFILES . '</a>';
```
Comment (drbyte) [inline includes/library/zencart/DashboardWidget/src/Logs.php]: href?

**Pull request merged**